package.json

```diff
@@ -53,7 +53,7 @@
   "scripts": {
     "lint": "eslint lib test",
     "pretest": "npm run lint",
-    "test": "NODE_ENV=test mocha --exit",
+    "test": "NODE_ENV=test mocha 'test/test_*'",
```

Comment:
Anyone running `npm test`, including CI, will still run all tests. But you can now also run `mocha test/path_to_file.js` to run a single test.